src/test/java/com/zerobank/pages/AccountSummary.java

```diff
@@ -10,7 +10,10 @@
     public void clickAccount(String str){
         WebElement webElement = Driver.get().findElement(By.xpath("//a[text()='"+ new StringUtils().capitalizeWord(str)  +"']"));
         webElement.click();
+    }
 
+    public String getTitle(){
+        return new AccountSummary().getTitle();
     }
```

Comment:
we could create a abstract method for this one cos we could use getTitle in every scenerio or Driver.get().getTitle() could be used in stepDefs